CHANGES.txt

```diff
@@ -44,6 +44,9 @@ Changes
 Fixes
 =====
 
+ - Fixed handling of null values (e.g. ``SELECT null FROM unnest([null])``
```

Comment:
this is a fix that only affects HTTP requests, you can write something like d1adc14186a5ba208747fe26c9b9233764032b6e